pkg/pgmodel/migrate.go

```diff
@@ -34,24 +35,21 @@ const (
 	truncateMigrationsTable     = "TRUNCATE prom_schema_migrations"
 
 	preinstallScripts = "preinstall"
-	versionScripts    = "versions"
+	versionScripts    = "versions/dev"
 	idempotentScripts = "idempotent"
-
-	// TODO: once we have a schema upgrade version for migration scripts,
-	// add tests and remove this flag
-	upgradeUntested = true
 )
 
 var (
 	ExtensionIsInstalled = false
 
-	toc = map[string][]string{
+	tableOfContets = map[string][]string{
 		"idempotent": {
 			"base.sql",
 			"matcher-functions.sql",
 		},
 	}
-	migrateMutex = &sync.Mutex{}
+	migrateMutex            = &sync.Mutex{}
+	migrationFileNameRegexp = regexp.MustCompile(`([[:digit:]]*)-[[:word:]]*.sql`)
```

Comment:
(note to self e.g. `6-foo.sql`, actually, should that be a comment?)
@cevian should this be
```
([[:digit:]]+)-[[:word:]]*.sql
```
do we use this for files that don't have a version?